todo-vue/src/components/Home.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="home">
+    <header><h1>Todo App</h1></header>
+    <main>
+      <ul>
+        <CardComponent
+          v-for="todo in todos"
+          :key="todo.id"
+          :title="todo.title"
+          :id="todo.id"
+          @setTitle="editTodo"
+          @delete="deleteTodo"
+        />
+      </ul>
+      <form @submit="addTodo">
+        <input id="new-todo-input" type="text" placeholder="add new todo..." />
```

Comment:
```suggestion
        <input id="new-todo-input" type="text" v-model="title" placeholder="add new todo..." />
```
data에 title을 정의해놓고, v-model로 매칭해주면 input에 값을 입력할 때 저절로 title에 값이 반영됩니다.
이런걸 "양방향 바인딩" 이라고 이야기해요!